solutions/fully-configurable/DA-types.md

```diff
@@ -0,0 +1,125 @@
+  - `skip_en_auth_policy` (optional): A boolean flag to determine whether to skip the creation of an authentication policy that allows Cloud Logs 'Event Source Manager' role access in the existing event notification instance. Defaults to `false`.
+  - `from_email` (optional): The `from` email address used in any Cloud Logs events coming via Event Notifications. Defaults to `cloudlogsalert@ibm.com`.
+  - `reply_to_email` (optional): The `reply_to` email address used in any Cloud Logs events coming via Event Notifications. Defaults to `no-reply@ibm.com`.
+  - `email_list` (optional): The list of email addresses to notify when Cloud Logs triggers an event. Defaults to an empty list.
+
+### Example Event Notification Instance Configuration
+
+```hcl
+cloud_logs_existing_en_instances = [
+  {
+    instance_crn        = "crn:v1:bluemix:public:...:event-notifications:instance"
+    integration_name    = "custom-logging-en-integration"
+    skip_en_auth_policy = true
+    from_email          = "examplefromemail@ibm.com"
+    reply_to_email      = "examplereplyemail@ibm.com"
+    email_list          = ["exampleemail1@ibm.com", "exampleemail2@ibm.com"]
+  },
+  {
+    instance_crn        = "crn:v1:bluemix:public:...:event-notifications:instance"
+    skip_en_auth_policy = false
+  }
+]
+```
+
+In this example:
+- The first EN instance has a integration name `"custom-logging-en-integration"` skips the authentication policy, and sets email notifications.
+- The second EN instance uses the default integration name and includes the authentication policy.
+
+
+## Cloud Logs Policies <a name="cloud_logs_policies"></a>
+
+The `cloud_logs_policies` input variable allows you to provide a list of policies that will be configured in the Cloud Logs service. Refer [here](https://cloud.ibm.com/docs/cloud-logs?topic=cloud-logs-tco-optimizer) for more information.
+
+- Variable name: `cloud_logs_policies`.
+- Type: A list of objects. Each object represents a policy.
+- Default value: An empty list (`[]`).
+
+### Options for cloud_logs_policies
+
+  - `logs_policy_name` (required): The unique policy name.
+  - `logs_policy_description` (optional): The description of the policy to create.
+  - `logs_policy_priority` (required): The priority to determine the pipeline for the logs. Allowed values are: type_unspecified, type_block, type_low, type_medium, type_high. High (priority value) sent to 'Priority insights' (TCO pipleine), Medium to 'Analyze and alert', Low to 'Store and search', Blocked are not sent to any pipeline.
+  - `application_rule` (optional): The rules to include in the policy configuration for matching applications.
+  - `subsystem_rule` (optional): The subsystem rules to include in the policy configuration for matching applications.
+  - `log_rules` (required): The log severities to include in the policy configuration.
+  - `archive_retention` (optional): Define archive retention.
+
+### Example cloud_logs_policies
+
+```hcl
+cloud_logs_policies = [
```

Comment:
The code block should not include `cloud_logs_policies =` in it. It should just include the value of the variable so users can click the copy to clipboard in the UI and paste the exact value